pdns/dnsdistdist/dnsdist.service.in

```diff
@@ -25,10 +25,10 @@ LimitNOFILE=16384
 # LimitMEMLOCK=infinity
 
 # Sandboxing
-# Note: adding CAP_SYS_ADMIN (or CAP_BPF for Linux >= 5.8) is required to use eBPF support,
```

Comment:
for the versions where CAP_BPF was enough, CAP_SYS_ADMIN also works?

Reply:
Yes, `CAP_BPF` is a subset of  `CAP_SYS_ADMIN`.